src/_includes/contributor/rewards.md

```diff
@@ -38,24 +51,24 @@ Every PR may receive several additional achievements during assessment.
+Advanced | 30 | Contribution provides new features, such as introducing a new CLI command or integration with the new payment or shipping methods.
+Complex | 20 | Contribution contains complete refactoring of legacy code, improvements to application design, or updates to libraries.
+Special achievement | 20 | Contribution earns recognition in a specific category, such as for extensive tests coverage, improved framework design, improved APIs or API coverage, and improved customizability.
+Test coverage | 10 | Contribution contains a fix or improvement and new tests or test cases.
+MFTF test coverage| 10 | Contribution contains MFTF tests.
+Bug fix | 10 | Contribution fixes one or more known GitHub issues.
+Author of Ported Issue | 5 | Additional points for a contribution that ports (up port or back port) a previous PR across release lines by another contributor
+Issue Triage: Confirmed | 5 | A public issue report is verified and confirmed. The description contains all the required information needed to easily reproduce the issue with the provided steps.
+Issue Triage: Rejected/Closed | 4 | A public issue report is verified and closed/rejected because it is not a bug or cannot be reproduce using the provided steps in the description on the supported Magento versions.
 
 ## DevDocs awards and points
 
-Contributors and Maintainers can also earn rewards for merged submissions to the [DevDocs repository](https://github.com/magento/devdocs). These earned points add to contributor totals. Like code contributions, the DevDocs team assesses the PRs and applies labels to determine the level of work and achievements. Each PR receives one base achievement and potentially additional achievements.
+Contributors and maintainers can also earn rewards for merged submissions to the [DevDocs repository](https://github.com/magento/devdocs). These earned points add to contributor totals. Like code contributions, the DevDocs team assesses the PRs and applies labels to determine the level of work and achievements. Each PR receives one base achievement and potentially additional achievements.
 
-See the following examples for calculated reward points:
+These are examples for calculated reward points:
 
--  Contributor submits PR with HTML formatting and typos fixes: Editorial(base) 1 point
--  Contributor submits PR with code sample update and new parameter descriptions: Improvement(base) 10 points + Major update(additional) 20 points = 30 points
+* A contributor who submits a PR with HTML formatting and typos fixes will receive an Editorial (base achievement) label = 1 point
+* A contributor who submits a PR with a code sample update and new parameter descriptions will receive an Improvement (base achievement) label =  10 points and a Major update (additional achievement) labe = 20 points for a total of 30 points
```

Comment:
Maybe we mention "Multiple minor editorial fixes should be rolled into a single PR, for the sake of efficiency." Watch that point grabbing!